src/components/field-book/badges.jsx

```diff
@@ -1,46 +1,39 @@
-import React, { PropTypes } from 'react';
-import { taxonBadges, multiBadges } from 'helpers/badge-groups';
-import { pluralize } from 'helpers/text';
+import React, { Component, PropTypes } from 'react';
+import { taxonBadges, multiBadges, crossBadges } from 'helpers/badge-groups';
 
-export const FieldBookBadges = ({ allWorkflows, activityByWorkflow }) => {
-  const taxons = taxonBadges(allWorkflows, activityByWorkflow);
-  const multis = multiBadges(activityByWorkflow);
-  return (
-    <div>
-      <h2>You have earned these badges</h2>
-        { Object.keys(taxons).sort().map((g, i) =>
-          <div className="badges" key={i}>
-          { taxons[g].map((b, j) =>
-              <div className="badge" key={j}>
-                <img src={require(`images/badges/${b.badge}`)} alt={`${b.name}`}></img>
-                <div className="description">
-                  {`The ${b.name} badge is earned for transcribing ${b.count} ${b.group}
-                    ${pluralize('records', b.count)}`}
-                </div>
-              </div>
-          )}
-          </div>
-        )}
-        { multis.length
-          ? <div className="badges">
-              { multis.map((b, j) =>
-                <div className="badge" key={j}>
-                  <img src={require(`images/badges/${b.badge}`)} alt={`${b.name}`}></img>
-                  <div className="description">
-                    {`The ${b.name} badge is earned for transcribing
-                      ${b.count} ${pluralize('records', b.count)}
-                      in ${b.expeditions} expeditions`}
-                  </div>
-                </div>
-              )}
+export class FieldBookBadges extends Component {
+  badges(earned, i = 1) {
```

Comment:
both `multis` and `cross` are rendered with index = 1, which I think React is having an issue with, preventing the render with `cross` badge(s)?

i think one possible solution - sending something like 'm' as the `i` for `multis` (line 27) and 'c' as the `i` for `cross` (line 28), and i think that would eliminate the need to set a default value for `i`?